Palace/Security/Obfuscator.swift

```diff
@@ -0,0 +1,78 @@
+// https://gist.github.com/DejanEnspyra/80e259e3c9adf5e46632631b49cd1007
+//  Obfuscator.swift
+//
+//  Created by Dejan Atanasov on 2017-05-31.
+//
+
+import Foundation
+
+class Obfuscator {
+    
+    // MARK: - Variables
+    
+    /// The salt used to obfuscate and reveal the string.
+    private var salt: String
+    
+    // MARK: - Initialization
+    
+    init() {
+        self.salt = "\(String(describing: TPPAppDelegate.self))\(String(describing: NSObject.self))"
+    }
+    
+    init(with salt: String) {
+        self.salt = salt
+    }
+
+    // MARK: - Instance Methods
+    
+    /**
+     This method obfuscates the string passed in using the salt
+     that was used when the Obfuscator was initialized.
+     
+     - parameter string: the string to obfuscate
+     
+     - returns: the obfuscated string in a byte array
+     */
+    func bytesByObfuscatingString(string: String) -> [UInt8] {
+        let text = [UInt8](string.utf8)
+        let cipher = [UInt8](self.salt.utf8)
+        let length = cipher.count
+        
+        var encrypted = [UInt8]()
+        
+        for t in text.enumerated() {
+            encrypted.append(t.element ^ cipher[t.offset % length])
+        }
+        
+        #if DEVELOPMENT
```

Comment:
Where is `DEVELOPMENT` defined? I couldn't find this constant in the project, but we have `DEBUG` for Debug mode

Reply:
This is borrowed code, this should be removed.